### PR TITLE
Fix Paragraph export

### PR DESCRIPTION
**Pull request by rin**
Importing the `Paragraph` component like we import the other components didn't work, because we were missing this …

### CODE REVIEW

File: src/lib/components/input/input.js [hunk 1]
```diff
@@ -1,83 +1,77 @@
...
-        {renderValidationError()}
+        <label htmlFor={name}>
+          <span className={styles.fieldName}>{label}</span>
+        </label>
+        <span className={`${styles.errorMessage} ${error && styles.active}`}>
+          {error}
+        </span>
+        {!error && helperText && (
+          <span className={styles.helperMessage}>{helperText}</span>
+        )}
```
Comment (davidpaley) [inline src/lib/components/input/input.js]: @rin why are you changing the input component in this PR too that is not related. Be careful because you are adding again the error with the validation that is solved in Staging

**Comment by rin** [comment]
@davidpaley Don't worry, I didn't touch anything except the Paragraph export, it's because this is an old PR that hasn't been rebased yet. Will rebase it now.

**Comment by rin** [comment]
@davidpaley Rebased!

**Comment by rin** [comment]
@davidpaley Rebased!
